libcrux-ml-kem/src/vector/avx2.rs

```diff
@@ -42,6 +42,218 @@ fn vec_from_i16_array(array: &[i16]) -> SIMD256Vector {
     }
 }
 
+#[inline(always)]
+#[hax_lib::fstar::verification_status(lax)]
+#[hax_lib::requires(fstar!("Spec.Utils.is_i16b_array (pow2 12 - 1) (repr $vector)"))]
+#[hax_lib::ensures(|out| fstar!("repr out == Spec.Utils.map_array (fun x -> if x >=. 3329s then x -! 3329s else x) (repr $vector)"))]
+fn cond_subtract_3329(vector: SIMD256Vector) -> SIMD256Vector {
+    SIMD256Vector {
+        elements: arithmetic::cond_subtract_3329(vector.elements),
+    }
+}
+
+#[inline(always)]
+#[hax_lib::fstar::verification_status(lax)]
+#[hax_lib::requires(fstar!("forall (i:nat). i < 16 ==> v (Seq.index (repr $vector) i) >= 0 /\\
+    v (Seq.index (repr $vector) i) < 3329"))]
+#[hax_lib::ensures(|out| fstar!("forall (i:nat). i < 16 ==> bounded (Seq.index (repr $out) i) 1"))]
+fn compress_1(vector: SIMD256Vector) -> SIMD256Vector {
+    SIMD256Vector {
+        elements: compress::compress_message_coefficient(vector.elements),
+    }
+}
+
+#[inline(always)]
+#[hax_lib::fstar::verification_status(lax)]
+#[hax_lib::requires(fstar!("(v $COEFFICIENT_BITS == 4 \\/
```

Comment:
You didn't want to start using string literal expressions to avoid escaping?

Reply:
Not yet, but yes, this should be done uniformly everywhere.

Reply:
ok did it. hopefully we can preserve it. am not sure if this is worth it considering that we would like to move to Rust annotations.